canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -157,6 +157,29 @@ class PropertiesMain extends React.Component {
 		this.initialValueInfo.additionalInfo.title = this.propertiesController.getTitle();
 	}
 
+	getApplyButtonLabel() {
+		if (this.props.propertiesConfig.buttonLabels) {
+			if (this.props.propertiesConfig.applyOnBlur && this.props.propertiesConfig.rightFlyout && this.props.propertiesConfig.buttonLabels.rejectLabel) {
```

Comment:
I think this should just return the applyLabel.  Maybe we should rename these properties to primary and secondary buttons to make it less confusing.  We don't need to change the internal code.